### PR TITLE
Continuing to streamline style usage

### PR DESCRIPTION
**Pull request by timea-solid**
This work concludes this Issue #533 :
* move almost all styles to style.js from each file. and fixed some typos in the process.  (point 4 on the issue list).
* streamline also styles in use with ns.ui('style') or ns.ui('background') (point 3 on the list)

Now we can start talking about changing the UI style.

### CODE REVIEW

**Review by SharonStrats — APPROVED** [review]
Very nice! 
my only thought is about the dist folder. I'm wondering if you should remove those files.

File: src/participation.ts [hunk 1]
```diff
@@ -31,10 +33,10 @@ const store = solidLogicSingleton.store as LiveStore
 *  @param {NamedNode} unused2/me - user that is logged into the pod (this argument is no longer used, but left in for backwards compatibility)
 *  @param {ParticipationOptions} options - the options that can be passed in are deleteFunction, link, and draggable these are used by the personTR button
```
Comment (TallTed) [inline src/participation.ts:34]:
```suggestion
*  @param {ParticipationOptions} options - the options that can be passed in are deleteFunction, link, and draggable; these are used by the personTR button
```

File: src/participation.ts [hunk 2]
```diff
@@ -192,7 +192,7 @@ export function recordParticipation (subject: NamedNode, padDoc: NamedNode, refr
 *
 *   @param {Document} dom  - the web page loaded into the browser
 *   @param {HTMLDivElement} container - the container element where the participants should be displayed
-*   @param {NamedNode} document - the document into which the particpation should be shown
+*   @param {NamedNode} document - the document into which the participation should be shown
 *   @param {NamedNode} subject - the thing in which participation is happening
 *   @param {NamedNode} me - the logged in user
 *   @param {ParticipationOptions} options - the options that can be passed in are deleteFunction, link, and draggable these are used by the personTR button
```
Comment (TallTed) [inline src/participation.ts:198]:
```suggestion
*   @param {ParticipationOptions} options - the options that can be passed in are deleteFunction, link, and draggable; these are used by the personTR button
```

File: src/styleConstants.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+export default {
+  highlightColor: '#7C4DFF', // Solid lavender https://design.inrupt.com/atomic-core/?cat=Core
+
+  formBorderColor: '#888888', // Mid-grey
+  formHeadingColor: '#888888', // originally was brown; now grey
+  lowProfileLinkColor: '#3B5998', // Grey-blue, e.g., for field labels linking to ontology
+  formFieldNameBoxWidth: '8em', // The fixed amount to get form fields to line up
+  // The latter we put in when switching awy from using tables.  Getting alignment between
+  // fields in different groups though is hard problem.
```
Comment (TallTed) [inline src/styleConstants.js:9]:
```suggestion
  // We put in the latter when switching away from using tables.  However, getting
  // alignment between fields in different groups is a hard problem.
```

File: src/styleConstants.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+  mediaModuleCanvasWidth: '640',
+  mediaModuleCanvasHeight: '480',
+
+  textInputSize: 20, // Default text input size in characters roughly
```
Comment (TallTed) [inline src/styleConstants.js:14]:
```suggestion
  textInputSize: 20, // Rough default text input size, in characters
```

File: src/widgets/dragAndDrop.js [hunk 5]
```diff
@@ -1,37 +1,32 @@
 /* Drag and drop common functionality
  *
  * It is easy to make something draggable, or to make it a drag target!
- * Just call the functions below.  In a solid world, any part of the UI which
- * represent one thing which has a UR, should be made draggable using makeDraggable
+ * Just call the functions below. In a solid world, any part of the UI which
```
Comment (TallTed) [inline src/widgets/dragAndDrop.js:4]:
```suggestion
 * Just call the functions below. In a Solid world, any part of the UI which
```

File: src/widgets/dragAndDrop.js [hunk 6]
```diff
@@ -1,37 +1,32 @@
...
+ * represent one thing which has a URI, should be made draggable using makeDraggable
  * Any list of things should typically allow you to drag new members of the list
```
Comment (TallTed) [inline src/widgets/dragAndDrop.js:6]:
```suggestion
 * represents one thing which has a URI, should be made draggable using makeDraggable.
 * Any list of things should typically allow you to drag new members of the list
```

File: src/widgets/dragAndDrop.js [hunk 7]
```diff
@@ -1,37 +1,32 @@
...
  * onto it.
- * The file upload function uploadFiles is provided as often of someone drags a file from the computer
+ * The file upload function uploadFiles is provided as often as someone drags a file from the computer
  * desktop, you may want to upload it into the pod.
```
Comment (TallTed) [inline src/widgets/dragAndDrop.js:9]:
```suggestion
 * The file upload function, uploadFiles, is provided as often as someone drags a file from the computer
 * desktop. You may want to upload it into the pod.
```

File: src/widgets/dragAndDrop.js [hunk 8]
```diff
@@ -1,37 +1,32 @@
...
 import * as debug from '../debug'
 import * as mime from 'mime-types'
+import * as style from '../style'
 
 /* global FileReader alert */
 
 export function makeDropTarget (ele, droppedURIHandler, droppedFileHandler) {
   const dragoverListener = function (e) {
-    e.preventDefault() // Neeed else drop does not work [sic]
+    e.preventDefault() // Need else drop does not work [sic]
```
Comment (TallTed) [inline src/widgets/dragAndDrop.js:19]:
```suggestion
    e.preventDefault() // Need this; otherwise, drop does not work.
```

**Comment by bourgeoa** [comment]
@timea-solid could you look at this issue ?
image

**Comment by bourgeoa** [comment]
This is the issue 
https://github.com/SolidOS/solid-ui/blob/878273708d800d65e33b0aec1568577c3df7e32b/src/tabs.ts#L185
and I don't understand why. The evaluation is not done correctly

I tried here and It does not work also when using our function

**Pull request merged**